### PR TITLE
Ui tests

### PR DESCRIPTION
**Pull request by TCourtneyOwen**
This PR isolates test code entirely from product code

### CODE REVIEW

**Comment by TCourtneyOwen** [comment]
I think I need to fix a merge conflict for the CI loop to initiate

**Comment by TCourtneyOwen** [comment]
The UI test fails on Mac because when we boot Excel we started the cached Office-Addin-Taskpane tests as opposed to the Custom Functions instance. presumably because both the manifests for the projects both have the same guids.  We need some better to way to ensure we aren't using cached add-ins

**Comment by TCourtneyOwen** [comment]
I think the right fix for the duplicate guids issue is to have the test call office-addin-manifest modifyManifestFile method to create a randomly generated guid before launching the taskpane app

**Comment by TCourtneyOwen** [comment]
I think I may have picked up a breaking change when merging master. Build now fails in CI loop with the following:
Error: ENOENT: no such file or directory, stat 'C:\agents\opx-azure-01\_work\1\s\dist\functions.json'
    at Object.statSync (fs.js:840:3)
    at C:\agents\opx-azure-01\_work\1\s\node_modules\custom-functions-metadata-plugin\lib\customfunctionsplugin.js:45:34
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\agents\opx-azure-01\_work\1\s\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:18:1)
    at AsyncSeriesHook.lazyCompileHook (C:\agents\opx-azure-01\_work\1\s\node_modules\tapable\lib\Hook.js:154:20)
    at Compiler.emitAssets (C:\agents\opx-azure-01\_work\1\s\node_modules\webpack\lib\Compiler.js:441:19)

**Comment by akrantz** [comment]
clean-webpack-plugin@3.0.0 does not work with custom-functions-metadata-plugin. Go back to using version 2.0.1.

File: package.json.orig [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
```
Comment (asachin96) [inline package.json.orig]: What is the purpose of this file?

Comment (TCourtneyOwen) [reply package.json.orig]: good catch - this should be removed - it gets created automatically during a merge and shouldn't have been included in the commit

**Comment by TCourtneyOwen** [comment]
the merge from Office-Addin-Taskpane looks to have introduced a failure in the CI loop that I am not seeing locally - investigating

**Comment by TCourtneyOwen** [comment]
I am going to close this PR.  I just opened a PR that squashes all commits and the CI loop passes on both Windows and Mac:  https://github.com/OfficeDev/Excel-Custom-Functions/pull/162

**Pull request closed without merging**